src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PrinterSettings.Windows.cs

```diff
@@ -517,7 +517,7 @@ public bool SupportsColor
         {
             get
             {
-                return GetDeviceCaps(Interop.Gdi32.DeviceCapability.BITSPIXEL, 1) > 1;
+                return DeviceCapabilities(SafeNativeMethods.DC_COLORDEVICE, IntPtr.Zero, 0) == 1;
```

Comment:
I agree with interpreting "capability not supported" as "color not supported".

Reply:
Is this comment referring that you agree with the new implementation?

Reply:
yes I agree with the new implementation.
Fax returns a failure code (-1) and indeed Fax does not support color